services/postgres/seed.ts

```diff
@@ -1,20 +1,61 @@
-import { PrismaClient } from "@prisma/client";
+import { PrismaClient } from "../../web-portal/backend/.generated/client";
 
 const prisma = new PrismaClient();
 
-// TODO: Initial seed data
+async function main() {
+  try {
+    await prisma.ruleType.createMany({
+      data: [
+        {
+          name: "Allowed Origins",
+          isEditable: true,
+          isMultiple: true,
+          validationType: "regex",
+          validationValue: `https?://(www\\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\\.[a-zA-Z0-9()]{1,6}\\b([-a-zA-Z0-9()!@:%_+.~#?&/\\//=]*)`,
+        },
+        {
+          name: "Approved Chains",
+          isEditable: true,
+          isMultiple: true,
+          validationType: "function",
+          validationValue: "Array.isArray(value)",
+        },
+        {
+          name: "Secret Key",
+          isEditable: false,
+          isMultiple: false,
+          validationType: "function",
+          validationValue: 'typeof value === "string" && value.length === 32',
+        },
+        {
+          name: "Allowed UserAgents",
+          isEditable: true,
+          isMultiple: true,
+          validationType: "regex",
+          validationValue: "*",
+        },
+        {
+          name: "Allowed Contracts",
```

Comment:
I don't think we have the ability to do this with our reliance on gateway kit. We aren't inspecting the contents of the RPC calls.